velodrome/sql/model.go

```diff
@@ -89,3 +90,15 @@ type Comment struct {
 	CommentUpdatedAt time.Time
 	PullRequest      bool
 }
+
+// Milestone contains information about github milestons and can be added to issues and prs
+type Milestone struct {
```

Comment:
Do we need all these fields?

Also do you expect this structure to be serialized? We don't have json annotation for other structures.